lib/rubocop/rspec/language.rb

```diff
@@ -5,6 +5,12 @@ module RSpec
     # RSpec public API methods that are commonly used in cops
     module Language
       RSPEC = '{(const {nil? cbase} :RSpec) nil?}'
+      BLOCK_PATTERN = <<-PATTERN
```

Comment:
Are those two used only once, to define a `def_node_matcher`? Maybe inline them?

Reply:
I would leave it until converting other RSpec keywords